tutorial-outline.md

```diff
@@ -0,0 +1,215 @@
+# Olmsted Screen Capture Tutorial Outline
+
+## Part 1: Olmsted Web Application Features (15-20 minutes)
+
+### 1.1 Accessing Olmsted (2-3 minutes)
+**Demo all three access methods:**
+- **Option A**: Public website at https://olmstedviz.org
+- **Option B**: Docker deployment (show command: `docker run -p 8080:3999 quay.io/matsengrp/olmsted`)
+- **Option C**: Local development server (show: `npm start` or `./bin/olmsted-server-local.sh`)
+
+### 1.2 Home Page & Dataset Management (3-4 minutes)
+**Show the splash page interface:**
+- Overview of the Available Datasets table
+- Upload methods:
+  - "Upload Data" button with file explorer
+  - Drag-and-drop functionality
+- Dataset selection checkboxes
+- Color-coded status indicators
+- Loading datasets into the query string
+- The "Explore!" button (emphasize need to select ≥1 dataset)
+
+### 1.3 Clonal Families Section - The Scatterplot (4-5 minutes)
+**Demonstrate interactive features:**
+- Locus selector (default: igh, or choose "ALL")
+- Axis configuration:
+  - X-axis: `unique_seqs_count` (default)
+  - Y-axis: `mean_mut_freq` (default)
+  - Show alternative field mappings
+- Point styling (color and shape customization)
+- **Faceting**: Split plot into panels by data field
+- **Interactions**:
+  - Click and drag to select subset of points
+  - Click individual points to filter
+
+### 1.4 Selected Clonal Families Table (2-3 minutes)
+**Show table features:**
+- Metadata columns display
+- Recombination event visualization
+- Sorting functionality
+- Selecting a clonal family (checkbox in "Select" column)
+- Automatic top family selection
+
+### 1.5 Clonal Family Details - Tree Visualization (4-5 minutes)
+**Demonstrate tree features:**
+- Phylogenetic tree display
+- Ancestral reconstruction method selector (if multiple available)
+- **Navigation**:
+  - Scroll to zoom (hover over tree)
+  - Click and drag to pan
+- **Alignment view**:
+  - Side-by-side with tree
+  - Color-coded amino acid mutations from root
+  - Vertical zoom synced with tree
+- **Leaf nodes**:
+  - Pie chart representation showing multiplicity
+  - Color-coded by sampling timepoint
+- **Configuration options** (below tree):
+  - Tree/alignment width ratio
+  - Field mapped to leaf size
+  - Maximum leaf size
+  - Tree tip labels toggle
+  - Branch width and color mappings
+
+### 1.6 Ancestral Sequences - Lineage View (2-3 minutes)
+**Demonstrate lineage drill-down:**
+- Click on a leaf label or circle
+- Show ancestral lineage alignment
+- Mutation highlighting from naive sequence
+
+### 1.7 Dataset Management in Visualization (1-2 minutes)
+**Show dynamic dataset switching:**
+- Dataset table on visualization page
+- "Update Visualization" button
+- "Manage Datasets" button (return to splash page)
+
+---
+
+## Part 2: Processing Your Own Data with olmsted-cli (15-20 minutes)
```

Comment:
Let's explain the bigger picture of the idea that olmsted has some custom requirements that we need to satisfy, hence we have a custom data format. OTOH we want to be able to uptake various data types.